template/footer.mustache

```diff
@@ -1,5 +1,80 @@
+            <a href='?{{{forms.get.page}}}={{{forms.pages.main}}}' class="link-1">Home</a>
+            
+            <a href='?{{{forms.get.page}}}={{{forms.pages.rules}}}'>How it works</a>
+                
+
+        </p>
+
+        <p class="footer-company-name">OneHourJam 2021</p>
+    </div>
+
+    <div class="footer-center">
+        <br>
+        <br>
+        Powered By - <br>
+        <a href='https://github.com/OneHourGameJam/OneHourGameJam'>One Hour Game Jam</a> version {{SOFTWARE_VERSION}} -
 <a href='https://github.com/OneHourGameJam/OneHourGameJam/issues' target='_BLANK'>Report a bug / suggest a feature</a>
-<br />
-{{{CONFIG.VALUES.SITE_FOOTER}}}
\ No newline at end of file
+
+    </div>
+
+    <div class="footer-right">
+
+        <p class="footer-company-about">
+            <span>About OneHourGameJam</span>
+            {{{CONFIG.VALUES.SITE_FOOTER}}}
+        </p>
+
+        <div class="footer-icons" id="mainmenu-social">
+            {{#CONFIG.VALUES.TWITTER_ACCOUNT}}
+                <div class="footer-icon-right">
+                    <a href="https://twitter.com/{{CONFIG.VALUES.TWITTER_ACCOUNT}}" target='_blank'>
+                        <div class='mediaButton twitterButton'><img src='{{{page.template_path}}}images/twitter.png' class="menuButton" /> Follow</div>
+                    </a>
+                </div>
+                {{/CONFIG.VALUES.TWITTER_ACCOUNT}}
+                {{#CONFIG.VALUES.DISCORD_INVITE_URL}}
+                <div class="footer-icon-right">
+                    <a href="{{{CONFIG.VALUES.DISCORD_INVITE_URL}}}" target='_blank'>
+                        <div class='mediaButton discordButton'><img src='{{{page.template_path}}}images/discord.png' class="menuButton" /> Discord</div>
+                    </a>
+                </div>
+                {{/CONFIG.VALUES.DISCORD_INVITE_URL}}
+                {{#CONFIG.VALUES.TWITCH_ACCOUNT}}
+                <div class="footer-icon-right">
+                    <a href="https://twitch.tv/{{CONFIG.VALUES.TWITCH_ACCOUNT}}" target='_blank'>
+                        <div class='mediaButton twitchButton'><img src='{{{page.template_path}}}images/twitch.png' class="menuButton" /> Watch</div>
+                    </a>
+                </div>
+                {{/CONFIG.VALUES.TWITCH_ACCOUNT}}
+
+                {{#CONFIG.VALUES.IRC_CHAT_IN_BROWSER}}
+                    {{#CONFIG.VALUES.IRC_ADDRESS}}
+                        {{#CONFIG.VALUES.IRC_CHANNEL}}
+                        <div class="footer-icon-right">
+                            <a href="{{{CONFIG.VALUES.IRC_CHAT_IN_BROWSER}}}" target='_blank'>
+                                <div class='mediaButton ircButton'><img src='{{{page.template_path}}}images/irc.png' class="menuButton" /> IRC</div>
+                            </a>
+                        </div>
+                            
+                        {{/CONFIG.VALUES.IRC_CHANNEL}}
+                    {{/CONFIG.VALUES.IRC_ADDRESS}}
```

Comment:
Indentation - should be in-line with `{{#CONFIG.VALUES.TWITTER_ACCOUNT}}` (Not required to fix, informational)
In this case this means one indent less for the entire block so that `{{#CONFIG.VALUES.IRC_CHAT_IN_BROWSER}}` is in line with `{{#CONFIG.VALUES.TWITTER_ACCOUNT}}` and then the rest of them have the same relative indnetation